pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -137,6 +146,65 @@
   blockchain_resources_03_desc: 'Try a crypto memory game created by Coinbase.'
   blockchain_resources_03_button_label: 'Try the Crypto Memory Game activity'
 
+  ai_hero_heading_01: "Artificial intelligence isn't magic…"
+  ai_hero_heading_02: "It's just code!"
+  ai_hero_desc: "Demystify artificial intelligence (AI) by learning how it's changing the ways we live, work, and learn."
+  ai_intro_heading: "Why learn about AI?"
+  ai_intro_video_title: "Why AI Matters with Deb Raji"
+  ai_intro_video_caption_link: "Check out the full Intro to AI series"
+  ai_intro_list_title_01: "Foundational knowledge"
+  ai_intro_list_desc_01: "Spark students' interest in technology and help them understand the world around them in a deeper way."
+  ai_intro_list_title_02: "Creativity and innovation"
+  ai_intro_list_desc_02: "Inspire students to think creatively and innovate, as they explore the potential applications of AI."
+  ai_intro_list_title_03: "Problem solving and critical thinking"
+  ai_intro_list_desc_03: "Encourage students to develop essential skills as they learn to design, build, and evaluate AI systems."
+  ai_why_heading: "Why teach AI with Code.org?"
+  ai_why_desc: "All the resources you need in one place! Explore our curriculum offerings, professional learning courses, and video series designed for you and your students."
+  ai_why_link_01: "Explore AI curricula"
+  ai_why_link_02: "See professional learning options"
+  ai_why_link_03: "Watch AI videos"
+  ai_curricula_heading: "AI Curricula"
+  ai_curricula_desc: "Explore the different AI curriculum offerings which meet diverse classroom needs."
+  ai_curriculum_title_01: "AI and Machine Learning"
+  ai_curriculum_desc_01: "Explore how computers learn from data. Develop machine learning projects around real-world data. Design machine learning app to solve personally relevant problem."
```

Comment:
Is this supposed to be `Design *a* machine learning app ...`?

Reply:
Yes 😁 (thank you for finding all the typos we missed haha!)